examples/crd-client/Makefile

```diff
@@ -25,7 +25,7 @@
 
 REPOSITORY ?= us-docker.pkg.dev/agones-images/examples
 
-server_tag = $(REPOSITORY)/crd-client:0.9
+server_tag = $(REPOSITORY)/crd-client:1.0
```

Comment:
@markmandel - how do you feel about going from 0.9 to 1.0 vs. 0.9 to 0.10? Are we happy calling this image a 1.0 image?

Reply:
That's a good point - I figure let's go to 0.10 -- examples can always change I guess (or at least we should be consistent with our examples).

Reply:
I've changed it to 0.10